app/views/root/400.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= render partial: "error_page", locals: {
     heading: "Sorry, there is a problem",
```

Comment:
Maybe out of scope for this PR but there's a lot of hard coded content in these templates. Could we translate it? I don't know if this content would ever actually be translated, I'm picturing a situation where we implement something to set locale based on user info or whatever. It's more for good practice. At the very least it'll handle a few instances of repeated content. I don't even know if static supports translations.

Reply:
I agree, there's a lot of content that could be translated here. I can't see `rails-i18n` or any localisation files in Static, so I think it'd need to be set up from scratch. With that in mind, setting up translations is outside of the scope of this pull request / should be a thing that is worthy of its own pull request.

I've raised this an an issue so we don't forget: https://github.com/alphagov/static/issues/2448

Reply:
That's perfectly reasonable. Is there a way we can at least make some of this content non-repeatable, maybe in a little partial or something? It's not a huge deal but _just in case_ it has to change suddenly we can update it once.

Not a blocker for merging. Just blasting opinions.

Reply:
I've had a go at this and I think it ends up convoluting things - the template ends up calling a partial, which calls another partial - and then the middle partial only sometimes overwrites the variables inside. It feels like premature optimisation that'd need to be undone when translations are used.

I think your first solution of using translations would be the best, simplest, and most future proof method to solve this - and that's outside the scope of this pull request.